app/controllers/members_controller.rb

```diff
@@ -54,6 +54,11 @@ def show
     end
   end
 
+  def destroy
```

Comment:
TODO!! Needs to check the password supplied is correct before letting them destroy their account.